.run/[Shared] run folder classes.run.xml

```diff
@@ -1,11 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Zou je deze twee bestanden (en die andere .xml beneden) weer opnieuw toe willen voegen? Die zijn op de een of andere manier verwijdert geraakt?